examples/ecs/event.rs

```diff
@@ -43,14 +42,9 @@ fn event_trigger_system(
     }
 }
 
-#[derive(Default)]
-struct EventListenerState {
-    my_event_reader: EventReader<MyEvent>,
-}
-
 // prints events as they come in
-fn event_listener_system(mut state: ResMut<EventListenerState>, my_events: Res<Events<MyEvent>>) {
-    for my_event in state.my_event_reader.iter(&my_events) {
+fn event_listener_system(mut state: Local<EventReader<MyEvent>>, my_events: Res<Events<MyEvent>>) {
```

Comment:
can we rename the `state` variable to `my_event_reader`?

Reply:
Yep